src/components/core/PivotTable.tsx

```diff
@@ -1046,7 +1047,10 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
             this.columnApi.setColumnWidth(column, this.autoResizedColumns[id].width);
         } else {
             await this.autoresizeColumnsByColumnId(this.columnApi, this.getColumnIds([column]));
-            this.resizedColumnsStore.addToManuallyResizedColumn(column);
+            if (isColumnDisplayed(this.columnApi.getAllDisplayedVirtualColumns(), column)) {
```

Comment:
my rule of thumb is avoiding nested IF statements, but it's optional :)

Reply:
done